src/panels/config/integrations/ha-integration-header.ts

```diff
@@ -150,18 +152,19 @@ export class HaIntegrationHeader extends LitElement {
       color: var(--secondary-text-color);
     }
     .icons {
```

Comment:
Should we add a `margin-left: 2px` so that letters will never glue against the left side before being wrapped?